R/bounds.r

```diff
@@ -1,3 +1,14 @@
+# If needed, converts input from classes that lack division and multiplication
+# to classes that can be scaled.
+# @param x vector of values to convert, if needed
+adapt_input_class <- function(x) {
```

Comment:
I'm not sure I follow the reasoning here. Why not just unconditionally coerce to numeric?